### PR TITLE
feat: deactivate formatting for other design

### PR DESCRIPTION
**Pull request by fanevasoa**

### CODE REVIEW

File: models/blog_design_command_factory.py
Comment (eojedapilchik) [inline models/blog_design_command_factory.py:1]: I think in this method can be simplified to something like:
```
class BlogDesignCommandFactory:
    EXAMPLE_PROMPT = "example"
    FAQ_CONTENT = "faq content"
    SPECIAL_BLOG_NAME = "kreatorcv.com"

    @staticmethod
    def create_command(blog_name: str, prompt_type: str):
        normalized_prompt_type = prompt_type.lower().strip()
        is_special_blog = blog_name.strip() == BlogDesignCommandFactory.SPECIAL_BLOG_NAME

        if normalized_prompt_type == BlogDesignCommandFactory.EXAMPLE_PROMPT:
            if is_special_blog:
                return ExampleDefaultPromptCommand()
            return CommonExamplePromptCommand()
        elif normalized_prompt_type == BlogDesignCommandFactory.FAQ_CONTENT:
            if is_special_blog:
                return FAQDefaultContentCommand()
            return CommonFAQContentCommand()
        
        return DefaultPromptCommand()
```

File: models/prompt_command_factory.py [hunk 1]
```diff
@@ -22,7 +22,7 @@ def create_command(prompt_type: str) -> PromptCommand:
             return SampleResumeExampleCommand()
         elif prompt_type in ["faq title"]:
             return FAQTitleCommand()
-        elif prompt_type in ["faq content"]:
-            return FAQContentCommand()
+        elif prompt_type.lower().strip() in ["faq content", "example"]:
+            return BlogDesignCommandFactory().create_command(blog_name, prompt_type)
```
Comment (eojedapilchik) [inline models/prompt_command_factory.py:26]: no need to instantiate the class you can just do (statically):

`return BlogDesignCommandFactory.create_command(blog_name, prompt_type)`

**Review by eojedapilchik — CHANGES_REQUESTED** [review]

File: models/blog_design_command_factory.py
Comment (fanevasoa) [reply models/blog_design_command_factory.py:1]: Special blog can be more than one blog so I'll do this like `SPECIAL_BLOG_NAMES = ["kreatorcv.com"]`

Comment (eojedapilchik) [reply models/blog_design_command_factory.py:1]: ok

Comment (fanevasoa) [reply models/blog_design_command_factory.py:1]: so the code will be like 
`class BlogDesignCommandFactory:
    EXAMPLE_PROMPT = "example"
    FAQ_CONTENT = "faq content"
    SPECIAL_BLOG_NAME = ["kreatorcv.com"]
    
    @staticmethod
    def create_command(blog_name: str, prompt_type:str) -> PromptCommand:
        normalized_prompt_type = prompt_type.lower().strip()
        is_special_blog = blog_name.strip() in BlogDesignCommandFactory.SPECIAL_BLOG_NAME

        if normalized_prompt_type == BlogDesignCommandFactory.EXAMPLE_PROMPT:
            if is_special_blog:
                return ExampleDefaultPromptCommand()
            return CommonExamplePromptCommand()
        elif normalized_prompt_type == BlogDesignCommandFactory.FAQ_CONTENT:
            if is_special_blog:
                return FAQDefaultContentCommand()
            return CommonFAQContentCommand()
        
        return DefaultPromptCommand()`

Comment (eojedapilchik) [reply models/blog_design_command_factory.py:1]: ok

**Review by eojedapilchik — APPROVED** [review]

**Pull request merged**